### ISSUE TITLE
Ability to retrieve a package with only the name

### ISSUE THREAD

**Issue by bbaker2** [kevinohara80/grunt-ant-sfdc#52]
The Ant Migration Tool allows you to specify a package name instead of a package.xml to retrieve _all_ components that belong to that package (as defined on SFDC's server)

Please see the [packageNames](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_deploying_ant_retrieveCode.htm) attribute during `sf:retrieve`

**Comment by loganm** [comment]
There's an error happening when a packageName is not specified.

```
Warning: An error occurred while processing a template (packageNames is not defined). Use --force to continue.
```

**Comment by loganm** [comment]
`if (myVar)` throws an error if myVar is undefined. Adding an undefined check fixes the problem. I'll open a PR to fix it tomorrow.

**Comment by codebaker2** [comment]
@loganm I'm having trouble reproducing. Can I get an example of your Gruntfile.js or run arguments?

**Comment by loganm** [comment]
All good. It's fixed in #57, thanks Ben.

**Comment by codebaker2** [comment]
Ah thanks. Sorry about the bug.
